src/main/java/net/catenax/semantics/triples/SparqlHelper.java

```diff
@@ -45,7 +45,7 @@
     /**
      * a regular expression to extract the command of the query
      */
-    protected final static Pattern COMMAND_PATTERN=Pattern.compile("(?<Command>SELECT|INSERT|DELETE)[^;]*WHERE\\s*\\{[^;]*\\}");
+    protected final static Pattern COMMAND_PATTERN=Pattern.compile("(?<Command>SELECT|INSERT|DELETE)[^;]*WHERE\\s*\\{[\\s\\S]*\\}");
```

Comment:
Hmm, yeah. The idea was that you may have several "commands" in a single "query" and those would be separated via semi-colon.
Guess I should have had a look at the SparQL spec a bit more thoroughly.